kubejs/server_scripts/tconstruct.js

```diff
@@ -147,4 +151,52 @@ ServerEvents.recipes((event) => {
             .mixing(Fluid.of(output, FluidAmounts.NUGGET), ingMix)
             .heated();
     }
+    
+    function addMMT(material_name, ingot_melt_ticks, ingot_cool_ticks, temperature, crushing_nuggets) {
+        let nugget_melt_ticks = ingot_melt_ticks / 9;
+        let block_melt_ticks = ingot_melt_ticks * 9;
+        let nugget_cool_ticks = ingot_cool_ticks / 9;
+        let block_cool_ticks = ingot_cool_ticks * 9;
```

Comment:
Does tinkers do this or do they have different times for nuggets and blocks?

Reply:
they have different times for block or nugget yes

Reply:
ok can we use does times then?

Reply:
huh?

Reply:
does tinkers construct use cooling/melting times for nuggets that are 9 times as fast as the cooling/melting times of ingots or do they have different times.

Reply:
yes
the reason it's still best to do is because it takes up less slots but yes that is how it works

Reply:
ok then we can use this. i was just wondering if that was the case or if it took less or more time to cool / melt a block than 9 ingots.

Reply:
like if it only took 8 times as much time to melt a block